src/Ch9/Ch9.Shared/ShowPage.xaml.cs

```diff
@@ -6,6 +6,7 @@
 using System.Runtime.InteropServices.WindowsRuntime;
 using Windows.Foundation;
 using Windows.Foundation.Collections;
+using Windows.Graphics.Display;
```

Comment:
Again, I'm not sure to understand why this file is changing.

Reply:
Missed it